### DISCUSSION TITLE
Equipos

### DISCUSSION THREAD

**Discussion by MarisaE-S** [Launch-X-Latam/MisionFrontEnd#549] [Team Posts]
Hola cómo saber si ya estas en un equipo?

**Comment by NelsonOrtizLeal** [comment]
Los equipos los haces tu o pides que te agreguen a uno. Los equipos se conforman por maximo 5 persona. Lo que algunos hacen es crear un grupo en teams y te agregan o el capitan del equipo puede que tenga otra forma de comunicarse. Sea por chats de teams o por alguno otro medio.
